examples/notebooks/bots_on_solidity.ipynb

```diff
@@ -721,7 +721,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.16"
+   "version": "3.10.10"
```

Comment:
in general when this is the only change to the notebook please try to exclude it from the PR

Reply:
fixed in 31d4132a9bce8021c0678d70d6a3b70d11fbf781